vault/audit.go

```diff
@@ -658,7 +658,7 @@ func (c *Core) ReloadAuditLogs() {
 	c.stateLock.RLock()
 	defer c.stateLock.RUnlock()
 
-	if c.Sealed() {
+	if c.Sealed() || (c.standby.Load() && !c.StandbyReadsEnabled()) || c.activeContext == nil {
```

Comment:
How is this change relevant to this PR?

Reply:
@JanMa I was re-evaluating SIGHUP reload strategy when I was adding support for it here, and realized the conditions here in audit was incorrect. I can change it if you want.